packages/main/src/pages/protected/workspace/Workspace.module.css

```diff
@@ -0,0 +1,149 @@
+#workspaceSidebar {
+  height: 95vh;
+  width: 70%;
+  background: #007b53;
+  color: #fff;
+  display: flex;
+  flex-direction: column;
+  /* align-items: center; */
+  /* justify-content: space-between; */
+  border: 1px solid #007b53;
+}
+
+#workspaceBox {
+  height: 75%;
+  padding: 0 10px 0 20px;
+  /* overflow-y: scroll; */
+}
+
+.workspaceInfo {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+}
+.workspaceWrap:first-child {
+  margin-top: 20px;
+}
+
+.workspaceWrap {
+  display: flex;
+  align-items: flex-end;
+  gap: 8px;
+}
+
+@media (min-width: 769px) {
+  #workspaceSidebar {
+    width: 70px;
+  }
+  .workspaceInfo,
+  .optionName {
+    display: none;
+  }
+
+  #workspaceBox {
+    padding: 0;
+    margin: 0 auto;
+  }
+
+  .workspaceAdd {
+    margin-bottom: 20px;
+    gap: 0;
+    justify-content: center;
+  }
+}
+
+
+.workspaceWrap h3,
+.workspaceWrap p {
+  margin-bottom: 0;
+  padding: 0;
+  line-height: 20px;
+}
+
+.workspaceAvatar {
+  border-radius: 10px;
+  border: 3px solid transparent;
+  text-align: center;
+  margin-bottom: 5px;
+  padding: 0.5px;
+}
+
+.workspaceAvatar:hover {
+  border: 3px solid #dddddd;
+}
+
+.currentWorkspace {
+  border-radius: 10px;
+  border: 3px solid #ddd;
+  text-align: center;
+  margin-bottom: 5px;
+  padding: 0.5px;
+}
+
+.workspaceAvatarM {
+  height: 40px;
+  width: 40px;
+  border-radius: 6px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: rgb(69, 64, 64);
+  color: #fff;
+  text-transform: uppercase;
+}
+
+/* #workspaceAdd {
+    height: 40px;
+    width: 40px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    border-radius: 10px;
+} */
```

Comment:
This block of commented out code can be removed if it's not being used

Reply:
This has been removed. it was not needed.